pkgs/applications/editors/mindforger/default.nix

```diff
@@ -0,0 +1,38 @@
+{ stdenv, fetchurl, ccache, qmake, qtbase, qtwebkit }:
+
+stdenv.mkDerivation rec {
+  name = "mindforger-${version}";
+  version = "1.48.2";
+
+  src = fetchurl {
+    url = "https://github.com/dvorka/mindforger/releases/download/1.48.0/mindforger_${version}.tgz";
```

Comment:
That looks weird, but it's not your fault. Upstream has a creative use of tags >;<